includes/Modules/Idea_Hub.php

```diff
@@ -146,12 +161,123 @@ function( $maybe_empty, $postarr ) {
 	 */
 	public function register() {
 		$this->register_scopes_hook();
+		if ( $this->is_connected() ) {
+			/**
+			 * Show admin notices on the posts page if we have saved / new ideas.
+			 */
+			add_filter( 'googlesitekit_admin_notices', $this->get_method_proxy( 'admin_notice_idea_hub_ideas' ) );
+		}
 
 		$this->post_name_setting->register();
 		$this->post_text_setting->register();
 		$this->post_topic_setting->register();
 	}
 
+	/**
+	 * Shows admin notification for idea hub ideas on post list screen.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $notices Array of admin notices.
+	 * @return array Array of admin notices.
+	 */
+	private function admin_notice_idea_hub_ideas( $notices ) {
+		global $post_type;
+		$current_screen = get_current_screen();
+		if ( is_null( $current_screen ) || 'edit-post' !== $current_screen->id || 'post' !== $post_type ) {
+			return $notices;
+		}
+		$transients               = new Transients( $this->context );
+		$dismissed_items_instance = new Dismissed_Items( $this->user_options );
+		if ( $dismissed_items_instance->is_dismissed( 'saved-ideas' ) && $dismissed_items_instance->is_dismissed( 'new-ideas' ) ) {
+			return $notices;
+		}
+		$dismissed_items = $dismissed_items_instance->get_dismissed_items();
+		$saved_ideas     = $transients->get( self::TRANSIENT_SAVED_IDEAS );
+		$new_ideas       = $transients->get( self::TRANSIENT_NEW_IDEAS );
+
+		$notices[] = new Notice(
+			'saved',
```

Comment:
The slug should be unique across all notices, so we should at least make it specific to the module. Similarly, we should probably use the same slug for the notice as we use for the dismissed item. How about `idea-hub_saved-ideas` and `idea-hub_new-ideas` ?

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/f96128adcfde0ffcb4f0e5f6271b129652121b8e